src/effects/effectslot.cpp

```diff
@@ -146,9 +147,7 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
         m_pControlNumParameters->forceSet(pEffect->numKnobParameters());
         m_pControlNumButtonParameters->forceSet(pEffect->numButtonParameters());
 
-        // Enabled is a persistent property of the effect slot, not of the
-        // effect. Propagate the current setting to the effect.
-        pEffect->setEnabled(m_pControlEnabled->get() > 0.0);
+        pEffect->updateEngineState();
```

Comment:
Couldn't the effect's enabled state be out of sync with the EffectSlot's enabled status? That's what the original line was doing here.